packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -183,6 +184,11 @@ export const esmPatientRegistrationSchema = {
           _description: 'Optional RegEx for testing the validity of the input.',
         },
       },
+      locationTag: {
+        _type: Type.String,
+        _default: null,
+        _description: 'For locations questions only. A tag used to filter and categorize locations.',
```

Comment:
```suggestion
        _description: 'Only for fields with "person attribute" type `org.openmrs.Location`. This filters the list of location options in the dropdown based on their location tag. By default, all locations are shown.',
```

Please check AI-generated text before committing. The description as written makes no sense in this context.